src/components/PostsList/PostsList.tsx

```diff
@@ -1,37 +1,45 @@
 import React from 'react';
 import './PostsList.scss';
+import { Post } from '../../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div className="PostsList">
-    <h2>Posts:</h2>
+type Props = {
+  posts: Post[],
+  selectedPostId: number,
+  selectPost: (value: number) => void
+};
 
-    <ul className="PostsList__list">
-      <li className="PostsList__item">
-        <div>
-          <b>[User #1]: </b>
-          sunt aut facere repellat provident occaecati excepturi optio
-        </div>
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Close
-        </button>
-      </li>
+export const PostsList: React.FC<Props> = ({ posts, selectedPostId, selectPost }) => {
+  const openPost = (postId: number) => {
+    if (postId === selectedPostId) {
+      selectPost(0);
+    } else {
+      selectPost(postId);
+    }
+  };
```

Comment:
this function does not only opening right? maybe another function name will be more informative